src/lua/digest.c

```diff
@@ -90,13 +91,174 @@ lua_pbkdf2(lua_State *L)
 	return 1;
 }
 
+/* CRC32 interface {{{ */
+
+int
+crc32_methods_result(lua_State *L)
+{
+	/* Get <self.value>. */
+	lua_getfield(L, 1, "value");
+	return 1;
+}
+
+int
+crc32_methods_update(lua_State *L)
+{
+	uint32_t crc32_begin, crc32_result;
```

Comment:
In Tarantool there is no need to declare variables in the start of block. You may just declare them below.

Here and below.